apps/files/src/components/FileDetails.vue

```diff
@@ -85,7 +85,11 @@ export default {
     },
 
     currentTab() {
-      return this.currentSidebarTab?.tab || this.defaultTab
+      if (this.currentTab) {
+        return this.currentTab.tab
```

Comment:
Is the `tab` field guaranteed to have a value? I guess so... but if not, then you would have to check that as well to have the same logic as before.

Reply:
It shouldn't happen that it would be undefined or null but anything can go wrong so I added it back. Better safe than sorry